packages/unit-testing/stubs/workbook/ConditionalFilter.js

```diff
@@ -0,0 +1,81 @@
+define(['DataMeasure', './CalculatedMeasure', './ChildNodesSelector', './DescendantOrSelfNodesSelector', './Expression', './PathSelector'], function (
+	DataMeasure,
+	CalculatedMeasure,
+	DescendantOrSelfNodesSelector,
+	PathSelector,
+	ChildNodesSelector
+) {
```

Comment:
Missing `./` in DataMeasure "injection".
Missing Expression parameter in function.
Missing DimensionSelector "injection".
Wrong order position for ChildNodesSelector.
"Injections" and function parameters should be ordered alphabetiacally.

Reply:
DimensionSelector still missing.